pkg/config/config.go

```diff
@@ -21,172 +21,141 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/sealerio/sealer/utils/platform"
+
+	"github.com/imdario/mergo"
+
 	"github.com/sirupsen/logrus"
 	"gopkg.in/yaml.v3"
 	k8sv1 "k8s.io/api/core/v1"
 	k8sYaml "sigs.k8s.io/yaml"
 
-	"github.com/sealerio/sealer/common"
 	v1 "github.com/sealerio/sealer/types/api/v1"
 	v2 "github.com/sealerio/sealer/types/api/v2"
 	"github.com/sealerio/sealer/utils/os"
 )
 
-/*
-config in Clusterfile:
-
-apiVersion: sealer.aliyun.com/v1alpha1
-kind: Config
-metadata:
-  name: redis-config
-spec:
-  path: etc/redis-config.yaml
-  data: |
-       redis-user: root
-       redis-passwd: xxx
-
-Dump will dump the config to etc/redis-config.yaml file
-*/
-
 const (
 	Merge     = "merge"
 	Overwrite = "overwrite"
 )
 
 type Interface interface {
-	// Dump Config in Clusterfile to the cluster rootfs disk
+	// Dump Configs from Clusterfile to the cluster rootfs
 	Dump(configs []v1.Config) error
 }
 
 type Dumper struct {
-	Configs []v1.Config
-	Cluster *v2.Cluster
+	rootPath string
```

Comment:
Please add some comment for rootPath. At least you should explain it is the rootPath for what? config, or the sealer rootPath?